### DISCUSSION TITLE
[Feature] Immich App Upload from Mobile

### DISCUSSION THREAD

**Discussion by mptr8956** [immich-app/immich#11905] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to have an option similar to the web interface as a guest: an option to add photos using the media picker rather than only from the timeline. 

Why? I created users for my parents. They should not use immich to backup their photos but use it to see & upload photos of their grandchildren only. 

it is possible in the web interface but not in the app. there you can only add photos that are already uploaded to the server. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7075
